GitCommands/AsyncLoader.cs

```diff
@@ -138,7 +136,7 @@ public Task<T> Load<T>(Func<CancellationToken, T> loadContent, Action<T> onLoade
                     }
                     if (token.IsCancellationRequested)
                     {
-                        return default(T);
+                        return default;
```

Comment:
:memo: Conversions of `default(T)` to `default` should be sent as a single pull request with no other changes (may need to build off another pull request)